### PR TITLE
Home

### PR DESCRIPTION
**Pull request by amemariee**

### CODE REVIEW

File: Android/app/src/main/res/layout/activity_register_user.xml [hunk 1]
```diff
@@ -94,20 +94,20 @@
...
-        android:layout_marginLeft="164dp"
-        android:visibility="visible"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintHorizontal_bias="0.017"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintVertical_bias="0.485"
-        tools:ignore="MissingConstraints" />
+<!--    <ProgressBar-->
```
Comment (cookiewho) [inline Android/app/src/main/res/layout/activity_register_user.xml:97]: Not sure why this is commented, I think this should stay the same

File: Android/app/src/main/java/com/example/rocketcorner/SaveSharedPreference.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package com.example.rocketcorner;
```
Comment (cookiewho) [inline Android/app/src/main/java/com/example/rocketcorner/SaveSharedPreference.java:1]: not sure what the point of this file is, maybe delete if not needed?

File: Android/app/src/main/res/layout/activity_register_user.xml (hunk 1 above)
Comment (cookiewho) [reply Android/app/src/main/res/layout/activity_register_user.xml:97]: if worry is progress bar always showing, i fixed that on main

**Review by cookiewho — CHANGES_REQUESTED** [review]
Once the changes are made, should be good

**Review by roberthuskins — APPROVED** [review]

**Review by MrJKappa — APPROVED** [review]
n o i c e

**Review by cookiewho — APPROVED** [review]

**Pull request merged**